assets/js/order-script.js

```diff
@@ -121,5 +125,20 @@ jQuery(document).ready(function($) {
 		
 	});
 
+	// save document
+	$( ".wpo-wcpdf-save" ).on("click", function() {
+		$( ".save_order" ).trigger( "click" );
```

Comment:
That saves the whole order. Shouldn't this work more like regerenate, only saving the invoice data?

Reply:
I thought about that, I just wanted you to see first before start working on that. But yes, I agree.